src/test/java/bio/terra/service/filedata/azure/blobstore/AzureBlobStorePdaoTest.java

```diff
@@ -175,6 +189,46 @@ public void testCopyFileWithSas() {
     assertThat("output is expected", fsFileInfo, samePropertyValuesAs(expectedFileInfo));
   }
 
+  @Test
+  public void testCopyFileWithGcsFile() {
+    UUID fileId = UUID.randomUUID();
+    fileLoadModel.sourcePath(SOURCE_GCS_PATH);
+
+    FSFileInfo expectedFileInfo = mockFileCopy(fileId);
+
+    FSFileInfo fsFileInfo =
+        dao.copyFile(
+            new Dataset().id(UUID.randomUUID()).predictableFileIds(false),
+            BILLING_PROFILE,
+            fileLoadModel,
+            fileId.toString(),
+            AZURE_STORAGE_ACCOUNT_RESOURCE,
+            TEST_USER);
+    assertThat("output is expected", fsFileInfo, samePropertyValuesAs(expectedFileInfo));
+    verify(gcsPdao, times(1)).getBlobFromGsPathNs(any(), eq(SOURCE_GCS_PATH), eq(null));
```

Comment:
If you wish, you can omit `times(1)` since that's the default verification.

Reply:
I ended up just removing the verify since just having a successful call is validation enough